core/src/main/scala/kafka/coordinator/transaction/TransactionMetadata.scala

```diff
@@ -98,13 +104,24 @@ private[transaction] object TransactionMetadata {
 }
 
 // this is a immutable object representing the target transition of the transaction metadata
-private[transaction] case class TransactionMetadataTransition(producerId: Long,
-                                                              producerEpoch: Short,
-                                                              txnTimeoutMs: Int,
-                                                              txnState: TransactionState,
-                                                              topicPartitions: immutable.Set[TopicPartition],
-                                                              txnStartTimestamp: Long,
-                                                              txnLastUpdateTimestamp: Long)
+private[transaction] case class TxnTransitMetadata(producerId: Long,
+                                                   producerEpoch: Short,
+                                                   txnTimeoutMs: Int,
+                                                   txnState: TransactionState,
+                                                   topicPartitions: immutable.Set[TopicPartition],
+                                                   txnStartTimestamp: Long,
+                                                   txnLastUpdateTimestamp: Long) {
+  override def toString: String = {
```

Comment:
Do we need to override case class' `toString`? I think they are auto-provided already?